game/consumers.py

```diff
@@ -1,7 +1,10 @@
 import json
+
+from asgiref.sync import sync_to_async, async_to_sync
 from channels.generic.websocket import AsyncWebsocketConsumer
+from channels.generic.websocket import WebsocketConsumer
```

Comment:
You can change previous line to `channels.generic.websocket import AsyncWebsocketConsumer, WebsocketConsumer`